app/src/main/java/com/example/checkpoint/MainActivity.kt

```diff
@@ -223,7 +225,9 @@ fun CheckpointHome(mapView: MapView){
         scope.launch {
             if (scaffoldState.bottomSheetState.isCollapsed) {
                 scaffoldState.bottomSheetState.expand()
-            } else {
+            }
+
```

Comment:
Attempted to write code for collapsing bottom sheet scaffold by clicking on the text. It works well for expanding currently, but I think clicking on the label will help it close.  This is another accessibility thing.

Reply:
I don't see any difference here, beyond the curlies being on another line.  ???